validator/slashing-protection/import_cli.go

```diff
@@ -0,0 +1,45 @@
+package slashingprotection
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/shared/cmd"
+	"github.com/prysmaticlabs/prysm/shared/fileutil"
+	"github.com/prysmaticlabs/prysm/validator/accounts/prompt"
+	"github.com/prysmaticlabs/prysm/validator/db/kv"
+	"github.com/prysmaticlabs/prysm/validator/flags"
+	slashingProtectionFormat "github.com/prysmaticlabs/prysm/validator/slashing-protection/local/standard-protection-format"
+	"github.com/urfave/cli/v2"
+)
+
+func ImportSlashingProtectionCLI(cliCtx *cli.Context) error {
+	protectionFilePath, err := prompt.InputDirectory(cliCtx, prompt.SlashingProtectionJSONPromptText, flags.SlashingProtectionJSONFileFlag)
+	if err != nil {
+		return errors.Wrap(err, "could not get slashing protection json file")
+	}
+	if protectionFilePath != "" {
```

Comment:
I think we should return error if `protectionFilePath == nil` 

That seems very concerning, right now it just returns `nil`